understanding/21/status-messages.html

```diff
@@ -8,6 +8,17 @@
 	<body>
 		<h1>Status Messages<br />
 		Understanding SC 3.2.6</h1>
+
+	<section id="brief">
+		<h2>In brief</h2>
+		<dl>
+			<dt>Objective</dt><dd>Make users aware of important changes in content</dd>
+			<dt>Author task</dt><dd>Let assistive technologies notify users about changes to the content that don't take focus</dd>
```

Comment:
current:
> Let assistive technologies notify users about changes to the content that don't take focus

suggested:
_Let assistive technologies notify users about changes to content that doesn't receive focus_

Reply:
The core of the phrase is "changes that don't take focus", not "content that doesn't receive focus".
For instance, it would be rephrased "content changes that don't take focus".
To go one step further, I'm going to make this "content notices"

Reply:
Notifications about notices looked silly, so I slipped in "status"

Reply:
@stevefaulkner if you have time, please also have a look at https://github.com/w3c/wcag/pull/3219 which I just created